lightwood/encoders/categorical/onehot.py

```diff
@@ -1,6 +1,8 @@
 import torch
-from lightwood.encoders.text.helpers.rnn_helpers import Lang
 import numpy as np
+from copy import deepcopy
+from scipy.special import softmax
+from lightwood.encoders.text.helpers.rnn_helpers import Lang
```

Comment:
Should we really reuse this here ? Isn't it better to create a helper that is inherited by `Lang` and get rid of things like the `UNCOMMON_TOKEN` deletion (which seems to be uncecessary for the purposes of this encoder) ?

Reply:
Notice that `Lang` usage isn't introduced here. I agree we could probably do this, however I'm not sure if it should be part of this PR as it may impact accuracy in the categorical AE.

Reply:
Ah, ok, my bad, will make an issue about this.